src/web/services/environmentVariables.tsx

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+
+import { backendError } from '../utils/apiError';
+
+export type EnvironmentVariable = {
+  baseUrl: string;
+  isDevelopment: boolean;
+};
+
+export async function GetEnvironmentVariables() {
+  try {
+    const result = await axios.get<EnvironmentVariable>(`/envars`, {
+      validateStatus: (status) => status === 200,
+    });
+    return result.data;
+  } catch (e: unknown) {
+    throw backendError(e, 'Could not get environment variables');
```

Comment:
Is it worth calling this out more explicitly in the UI rather than just a backendError?

Reply:
I don't think it is really worth it:
* I think the endpoint is very safe, it just returns some strings which have fallback values in `envars.ts`. 
* The `.env` file would have to already be created to reach the UI, so I'm not sure what the error would actually say?